### PR TITLE
As/meta application fixes

### PR DESCRIPTION
**Pull request by magnearun**
Fixes to the Meta application

- Fetch organizations and ministries from Contentful and display in a Select field.
- Display repeater data in table

This also moves the Table component that was used in the ParentalLeave application into the shared lib, this component might belong in the ui library but will be here until that is decided. 

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against master before asking for a review

### CODE REVIEW

File: libs/application/templates/meta-application/src/fields/DataTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  const emptyMessage = formatText(
+    isDraft ? m.dataEmptyDraft : m.dataEmpty,
+    application,
+    formatMessage,
+  )
+
+  const data = useMemo(
+    () => (application as { answers?: { data?: Data[] } }).answers?.data ?? [],
+    [application.answers?.data],
```
Comment (arnifreyrs) [inline libs/application/templates/meta-application/src/fields/DataTable/index.tsx:41]: answers can never by undefined

File: libs/application/templates/meta-application/src/fields/DataTable/index.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  const isDraft = application.state === 'draft'
+
+  const emptyMessage = formatText(
+    isDraft ? m.dataEmptyDraft : m.dataEmpty,
+    application,
+    formatMessage,
+  )
+
+  const data = useMemo(
+    () => (application as { answers?: { data?: Data[] } }).answers?.data ?? [],
```
Comment (arnifreyrs) [inline libs/application/templates/meta-application/src/fields/DataTable/index.tsx:40]: I do not understand this useMemo, what are you trying to achieve?

could you do: `const data = application.answers.data ?? []` instead ?

**Review by arnifreyrs — APPROVED** [review]
lgtm!

File: libs/application/templates/meta-application/src/fields/DataTable/index.tsx (hunk 2 above)
Comment (magnearun) [reply libs/application/templates/meta-application/src/fields/DataTable/index.tsx:40]: React-table requires the data to be memoized https://react-table.tanstack.com/docs/api/useTable

Comment (arnifreyrs) [reply libs/application/templates/meta-application/src/fields/DataTable/index.tsx:40]: ok makes sense

**Pull request merged**